### PR TITLE
[emacs/en] Basic Emacs tutorial

### PR DESCRIPTION
**Pull request by Joseph-Riad**
- [X] I solemnly swear that this is all original content of which I am the original author
- [X] Pull request title is prepended with `[language/lang-code]`
- [X] Pull request touches only one file (or a set of logically related files with similar changes made)
- [X] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [X] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [X] Yes, I have double-checked quotes and field names!

A tutorial on the basics of Emacs. It can be enhanced by adding a primer on Emacs Lisp. I might add that later, but I'd prefer it if someone else contributed it as my experience with Elisp is limited right now.

### CODE REVIEW

File: emacs.html.markdown [hunk 1]
```diff
@@ -0,0 +1,320 @@
+---
+tool: emacs
```
Comment (divayprakash) [inline emacs.html.markdown]: Before the ```tool: emacs``` line, please add the following line

```
category: tool
```

Comment (Joseph-Riad) [reply emacs.html.markdown]: Done. Sorry I missed that.

**Comment by divayprakash** [comment]
@adambard Same issue as #3552 here. The file is built properly and `learnxinyminutes/com/docs/emacs` works perfectly but it doesn't show up in the `Tools` section. Any updates to the `Languages` section are working but not `Tools`. Please take a look.

**Comment by divayprakash** [comment]
> @adambard Same issue as #3552 here. The file is built properly and `learnxinyminutes/com/docs/emacs` works perfectly but it doesn't show up in the `Tools` section. Any updates to the `Languages` section are working but not `Tools`. Please take a look.

Solved this. Just removed the `lang: en-en` line and it worked!

**Pull request merged**